packages/@n8n/task-runner/src/js-task-runner/js-task-runner.ts

```diff
@@ -542,4 +535,12 @@ export class JsTaskRunner extends TaskRunner {
 			...additionalProperties,
 		});
 	}
+
+	private vmCodeWrapper(code: string) {
+		return [
+			'globalThis.global = globalThis',
+			'Object.getPrototypeOf = () => ({}); Reflect.getPrototypeOf = () => ({}); Object.setPrototypeOf = () => false; Reflect.setPrototypeOf = () => false',
```

Comment:
Since we join these this could be split onto multiple lines and add a clarifying comment why these are here

```suggestion
			// Prevent prototype manipulation
			'Object.getPrototypeOf = () => ({})',
			'Reflect.getPrototypeOf = () => ({})',
			'Object.setPrototypeOf = () => false',
			'Reflect.setPrototypeOf = () => false'
```